### DISCUSSION TITLE
SESSION 5 - Start with Timelapse Night Photo

### DISCUSSION THREAD

**Discussion by ZontiSenior** [gopro/labs#689] [Q&A]
Hi, I can't manage to start the Session 5 with GoPro Labs in time-lapse night photo mode. Maybe someone can tell me which command to use. Thank you and best regards from Austria

**Comment by dnewman-gpsw** [comment]
Labs for Session 5 is very much incomplete. It doesn't have a full command set, and was intended only for video modes with Protune settings.

**Comment by ZontiSenior** [comment]
After many attempts I have now found a solution ;-)
With the command "mTN" the Session 5 camera starts in time-lapse night photo mode :-)
But it's true, unfortunately many things don't work with the Session 5.
